tests/topotests/bgp_minimum_holdtime/r1/zebra.conf

```diff
@@ -0,0 +1,9 @@
+!
+interface lo
+ ip address 172.16.255.254/32
```

Comment:
nit: This is not needed for this test, as I see.

Reply:
Fixed in b042667